app/assets/stylesheets/components/helpers/_variables.scss

```diff
@@ -0,0 +1,25 @@
+// Forked from GOV.UK Frontend, namespace changed to ensure no conflicts.
+
+$gem-spacing-scale-0: 0;
```

Comment:
Would you expect an app in a component to also use these variables? eg Spacing is a common thing.

Reply:
Hmm, that's a good point.

For now since these are forked lets keep them in here, the main reason to namespace these is to avoid collisions when importing GOV.UK Frontend, I'd imagine long term using the proper GOV.UK Frontend scale variables within downstream components is OK?